documentation/publish.md

```diff
@@ -63,7 +63,7 @@ Once your extension is signed, it is subject to review by Mozilla at any time. T
 
 There are four ways you can distribute your extension. They are:
 
-- Use [addons.mozilla.org](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/Distribution/Submitting_an_add-on) (most popular, broadest audience)
+- Use [addons.mozilla.org](https://developer.mozilla.org/docs/Mozilla/Add-ons/Distribution/Submitting_an_add-on) (most popular, broadest audience)
```

Comment:
Please change link to `/documentation/publish/submitting-an-add-on/`.

Reply:
Oops just noticed, is text quite clear? 

Use addons.mozilla.org (most popular, broadest audience) and we are linking amo to submitting-an-add-on.

Just cross checking

Reply:
The last one for you @caitmuenster ;)